unitTest/PageParserTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+include_once '../src/parserPages/PageParser.php';
+include_once '../src/dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+    public function testPageParser()
+    {
+        $objectPage = new PageParser();
+        $sourceLink = 'https://usn.ubuntu.com/usn/rss.xml';
+        $haystack = $objectPage->getValuesInItem($sourceLink);
+        $this->assertIsArray($haystack);
+        $this->assertEquals(10, count($haystack));
```

Comment:
This might be wrong when the number of items on remote resource will be changed. You should not test the number of items, but you can use mocks instead.